web/src/components/dashboard/SidebarCardTaskProgress.tsx

```diff
@@ -33,9 +35,13 @@ export const SidebarCardTaskProgress = (props: Props): ReactElement => {
   const { Config } = useConfig();
   const { userData } = useUserData();
   const { roadmap } = useRoadmap();
+  const modifiedRoadmap = LookupOperatingPhaseById(userData?.profileData.operatingPhase)
+    .displayBusinessStructurePrompt
+    ? roadmapWithSectionSpecificTasks(roadmap as Roadmap, "PLAN")
```

Comment:
I would rather make this a function or memo and add guard clauses if it is having issues with unfeined. Or just have your function allow and handle undefined values. Using `as` basically wipes away *any* typechecking that typescript would give us, and can lead to completely hiding away errors that SHOULD be occurring if the types change.